src/geocoder.ts

```diff
@@ -119,12 +124,12 @@ function geocodeChunk(
       return;
     }
 
-    const mergeRows: { [key: string]: any } = {};
+    const geocodeOutputs: { [key: string]: any } = {};
 
     for (let i = 0; i < body.length; i++) {
       const address = body[i];
       if (address.candidate_index === 0) {
-        mergeRows[address.input_index] = structuredRow(options.structure, address, options.columnPrefix, options.columnSuffix);
+        geocodeOutputs[geocodeInputsToRowsMap[address.input_index]] = structuredRow(options.structure, address, options.columnPrefix, options.columnSuffix);
```

Comment:
I feel like something is going on with the types here. We have:

```ts
const geocodeInputsToRowsMap: { [key: number]: number } = {}
```

...but also:

```ts
const geocodeOutputs: { [key: string]: any } = {};
```

So `geocodeInputsToRowsMap[...]` returns a `number`, right, but `geocodeOutputs` is keyed using strings? How did TypeScript pass that? Also, what's the type of `address.input_index`? I'm guessing the old code thinks it must be a `string`? I'm confused.